server/repository/src/db_diesel/invoice_row.rs

```diff
@@ -222,7 +224,7 @@ impl<'a> OutboundShipmentRowRepository<'a> {
             .filter(
                 type_
                     .eq(InvoiceRowType::OutboundShipment)
-                    .and(name_id.eq(name)),
+                    .and(name_link_id.eq(name)),
```

Comment:
This seems suss to me - should really be on `name_link::name_id` in some manner and this doesn't seem like that, will have to look again in rust analyser...

All that said, the only place this function is called is in a test anyway, but I can see need to see all outbound shipments for a particular customer/patient. Though... I am curious why there is no equivalent `InboundShipmentRowRepository`? Is `OutboundShipmentRowRepository` actually not used outside of tests? Can we just remove it? I have a suspicion all shipments are just queried through the base invoice repository really.